src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Input/KeyConverter.cs

```diff
@@ -4,246 +4,326 @@
 
 //
 //
-// Description:
+// Description: KeyConverter - Converts a Key string
+// to the *Type* that the string represents and vice-versa
 //
-//      KeyConverter : Converts a key string to the *Type* that the string represents and vice-versa
-//
-// Features:
-//
-//
-//
-// 
 
-using System;
 using System.ComponentModel;    // for TypeConverter
 using System.Globalization;     // for CultureInfo
-using System.Reflection;
-using System.Windows;
-using System.Windows.Input;
-using System.Windows.Markup;
-using MS.Utility;
+
 using MS.Internal.WindowsBase;
 
 namespace System.Windows.Input
 {
     /// <summary>
-    /// Key Converter class for converting between a string and the Type of a Key
```

Comment:
Since you're already updating the XML doc, could you use the docs from [learn.microsoft.com](https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.keyconverter) instead ? learn.microsoft.com is the source of truth so we should use it. If you want to modify the documentation you should open a PR in https://github.com/dotnet/dotnet-api-docs.

Reply:
`Converts a Key object to and from other types.`
vs.
`Converter class for converting between a string and Key`. (could be `conversion`, right)

While I agree with you about the source of truth, looking at the source, you clearly know this only works between `string` and `Key`, so the msdn description is less technical, and while it is correct in the spirit of what the class does, it doesn't specify it for the developer the implementation reality.

To be honest, I don't think I really wanna spend my time writing PRs for docs unless there's something horribly wrong, while noble and helpful, I'd do nothing else. But if I'm doing a PR on a small class, I want to have it right for others.